content/learn/node-difference-browser.fr.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Differences between Node.js and the Browser
```

Comment:
Looks like the titles and descriptions might have gotten missed in the translation pass

Reply:
Oh yess but title can't be translated ?

Reply:
Sorry, I'm not familiar with the i18n library used.
@ovflowd I've seen title vs. displayTitle on some of the other PRs, but what are translators supposed to do for these metadata items?

Reply:
my understanding is that we do not have to reduce category and section. for title I am not sure

Reply:
From the codebase, the title's should be the same, in case you want to offer same links for different languages. (https://github.com/nodejs/nodejs.dev/blob/main/src/templates/learn.tsx#L50)

But I forgot to actually update the learn pages doing the same thing on the remaining markdown pages of (`displayTitle`/ `title`).

I'm going to make a PR for this, so @AugustinMauroy if you could wait for that, that'd be great! 🙇

Reply:
@ovflowd 
I can wait, I have time. I'm still in school